src/figury/AnimatorApp.java

```diff
@@ -3,13 +3,8 @@
 import java.awt.EventQueue;
 import java.awt.Toolkit;
 
-import javax.swing.JFrame;
-import javax.swing.JPanel;
-import javax.swing.SwingUtilities;
-import javax.swing.JButton;
+import javax.swing.*;
```

Comment:
to jest na tyle duży pakiet, że importowanie wszystkiego zamiast wybranych klas nie jest korzystne